src/wallet/walletdb.cpp

```diff
@@ -419,7 +419,7 @@ ReadKeyValue(CWallet* pwallet, CDataStream& ssKey, CDataStream& ssValue,
                 return false;
             }
         } else if (strType != "bestblock" && strType != "bestblock_nomerkle" &&
-                strType != "minversion" && strType != "acentry") {
+                strType != "minversion" && strType != "acentry" && strType != "setting") {
```

Comment:
Note that #15588 is already proposing to change the same line to:

```cpp
strType != "minversion" && strType != "acentry" && strType != "version") {
```

Reply:
If that commit gets pulled I'll update this PR if it's not already pulled or closed.